From 906c826826308b8cd342e0eee8821cade728cb22 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 21 May 2021 23:02:56 +0200
Subject: [PATCH 15/N] Remove sudo

Currently we don't have sudo and most people have their MSYS2 roots
user-writable anyway, so let's completely remove the usage of sudo.  Most of the
patching is currently done in 0000-pacman-msysize.patch, these are just some new
occurrences.  Ideally this removal will be a completely separate patch in the
future.
---
 scripts/libmakepkg/executable/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/libmakepkg/executable/meson.build b/scripts/libmakepkg/executable/meson.build
index d5b6dcc..535da57 100644
--- a/scripts/libmakepkg/executable/meson.build
+++ b/scripts/libmakepkg/executable/meson.build
@@ -8,7 +8,6 @@ sources = [
   'gzip.sh.in',
   'pacman.sh.in',
   'strip.sh.in',
-  'sudo.sh.in',
   'vcs.sh.in',
 ]
 
-- 
2.33.0

